### ISSUE TITLE
Breakpoints not disappearing when removed

### ISSUE THREAD

**Issue by Rageoholic** [realgud/realgud#180]
When I set a breakpoint and remove it after I'm done with it with gdb, the breakpoint indicator remains even though there is no breakpoint there.

**Comment by Rageoholic** [comment]
Testing a little more, realgud:cmd-clear works fine. realgud:cmd-delete was causing the problem.

**Comment by rocky** [comment]
What debugger is running underneath? Does it issue status of deletion, e.g. "Breakpoint x deleted"? 
Does this happen across the board or just under some debuggers ?

**Comment by Rageoholic** [comment]
GDB. I haven't tested in other debuggers

**Comment by rocky** [comment]
Yeah, that is the problem. realgud tracks messages from the underlying debuggers. So no output, no tracking. 

It has been suggested that realgud:cmd-delete should somehow leave a breadcrumb around to note that if there isn't an indication to the contrary on output, e.g. no output, then it should assume the command succeeded.  I'd like a volunteer to address. I don't think I'll have the time in the near future, it doesn't bother me that yet (perhaps I don't use gdb enough),  and well, this _is_ open source.

**Comment by eflanigan00** [comment]
I see the same problem. gdb commands shows `delete 4` but the debugger shows no output.
It seems like realgud should know this because it is issuing the command to gdb, correct?

**Comment by rocky** [comment]
Ok. Perhaps you would care to implement a fix?

**Comment by sakurawald** [comment]
Still see the same bug, the `breakpoint mark in source buffer` should be removed.

**Comment by rocky** [comment]
Fixed in #324?
